kubernetes-discovery-client/src/main/java/io/micronaut/kubernetes/util/KubernetesUtils.java

```diff
@@ -58,10 +57,7 @@ public class KubernetesUtils {
 
     public static final String ENV_KUBERNETES_SERVICE_HOST = "KUBERNETES_SERVICE_HOST";
     private static final Logger LOG = LoggerFactory.getLogger(KubernetesUtils.class);
-    private static final List<PropertySourceReader> PROPERTY_SOURCE_READERS = Arrays.asList(
-            new YamlPropertySourceLoader(),
```

Comment:
are `YamlPropertySourceLoader`. and `JsonPropertySourceLoader` no longer necessary?

Reply:
This change is from `master` branch